### PR TITLE
Return error when the config checking is failed in the controller cmd

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: cmd/controller/app/server.go [hunk 1]
```diff
@@ -65,9 +65,11 @@ func NewControllerManagerCommand() *cobra.Command {
 		Use:   "controller-manager",
 		Short: `KubeSphere DevOps controller manager`,
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-			if errs := s.Validate(); len(errs) == 0 {
-				err = Run(s, signals.SetupSignalHandler())
+			if errs := s.Validate(); len(errs) != 0 {
+				return utilerrors.NewAggregate(errs)
 			}
+
+			err = Run(s, signals.SetupSignalHandler())
```
Comment (JohnNiang) [inline cmd/controller/app/server.go:72]: IMO, the code below could simplify `return err = Run(s, signals.SetupSignalHandler())`.

```go
err = Run(s, signals.SetupSignalHandler())
return
```

Comment (LinuxSuRen) [reply cmd/controller/app/server.go:72]: I didn't see the big differences.

Comment (JohnNiang) [reply cmd/controller/app/server.go:72]: Clear and consistent are big differences.

**Pull request merged**